doc/source/whatsnew/v0.24.0.txt

```diff
@@ -181,6 +181,7 @@ Reshaping
 Other
 ^^^^^
 
--
+- :meth:`Series.combine()` works correctly with :class:`~pandas.api.extensions.ExtensionArray` inside of :class:`Series` (:issue:`20825`)
```

Comment:
can you move the ExtensionArray to a sub-section (create a new one) as lots of changes in EA (you can make a new bug fix section).

Reply:
I just made a bug fix section